keyboards/theone/info.json

```diff
@@ -0,0 +1,251 @@
+            {"matrix": [2, 9], "x": 95, "y": 20, "flags": 4},
+            {"matrix": [2, 10], "x": 105, "y": 20, "flags": 4},
+            {"matrix": [2, 11], "x": 115, "y": 20, "flags": 4},
+            {"matrix": [2, 12], "x": 125, "y": 20, "flags": 4},
+            {"matrix": [2, 13], "x": 135, "y": 20, "flags": 4},
+            {"matrix": [1, 16], "x": 152, "y": 20, "flags": 4},
+            {"matrix": [2, 16], "x": 152, "y": 30, "flags": 4},
+            {"matrix": [3, 13], "x": 127.5, "y": 30, "flags": 4},
+            {"matrix": [3, 11], "x": 117.5, "y": 30, "flags": 4},
+            {"matrix": [3, 10], "x": 107.5, "y": 30, "flags": 4},
+            {"matrix": [3, 9], "x": 97.5, "y": 30, "flags": 4},
+            {"matrix": [3, 8], "x": 87.5, "y": 30, "flags": 4},
+            {"matrix": [3, 7], "x": 77.5, "y": 30, "flags": 4},
+            {"matrix": [3, 6], "x": 67.5, "y": 30, "flags": 4},
+            {"matrix": [3, 5], "x": 57.5, "y": 30, "flags": 4},
+            {"matrix": [3, 4], "x": 47.5, "y": 30, "flags": 4},
+            {"matrix": [3, 3], "x": 37.5, "y": 30, "flags": 4},
+            {"matrix": [3, 2], "x": 27.5, "y": 30, "flags": 4},
+            {"matrix": [3, 1], "x": 17.5, "y": 30, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 30, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 40, "flags": 4},
+            {"matrix": [4, 2], "x": 22.5, "y": 40, "flags": 4},
+            {"matrix": [4, 3], "x": 32.5, "y": 40, "flags": 4},
+            {"matrix": [4, 4], "x": 42.5, "y": 40, "flags": 4},
+            {"matrix": [4, 5], "x": 52.5, "y": 40, "flags": 4},
+            {"matrix": [4, 6], "x": 62.5, "y": 40, "flags": 4},
+            {"matrix": [4, 7], "x": 72.5, "y": 40, "flags": 4},
+            {"matrix": [4, 8], "x": 82.5, "y": 40, "flags": 4},
+            {"matrix": [4, 9], "x": 92.5, "y": 40, "flags": 4},
+            {"matrix": [4, 10], "x": 102.5, "y": 40, "flags": 4},
+            {"matrix": [4, 11], "x": 112.5, "y": 40, "flags": 4},
+            {"matrix": [4, 13], "x": 122.5, "y": 40, "flags": 4},
+            {"matrix": [4, 14], "x": 142, "y": 42, "flags": 4},
+            {"matrix": [5, 15], "x": 152, "y": 52, "flags": 4},
+            {"matrix": [5, 14], "x": 142, "y": 52, "flags": 4},
+            {"matrix": [5, 13], "x": 132, "y": 52, "flags": 4},
+            {"matrix": [5, 12], "x": 120, "y": 50, "flags": 4},
+            {"matrix": [5, 10], "x": 110, "y": 50, "flags": 4},
+            {"matrix": [5, 9], "x": 100, "y": 50, "flags": 4},
+            {"matrix": [5, 6], "x": 37.5, "y": 50, "flags": 4},
+            {"matrix": [5, 2], "x": 25, "y": 50, "flags": 4},
+            {"matrix": [5, 1], "x": 12.5, "y": 50, "flags": 4},
+            {"matrix": [5, 0], "x": 0, "y": 50, "flags": 4}
+        ],
+        "max_brightness": 130,
+        "speed_steps": 52,
+        "val_steps": 26
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x3245",
+        "vid": "0x19F5"
```

Comment:
This Vendor ID is in use: 
https://www.the-sz.com/products/usbid/index.php?v=0x19F5&p=&n=

Generally, above 0x4000 is best to use, and the further past it, the better.

Reply:
Thanks, the Vendor ID has been edited to 0x49F5.